scout/demo/cancer.load_config.yaml

```diff
@@ -44,9 +44,7 @@ delivery_report: scout/demo/delivery_report.html
 cnv_report: scout/demo/cancer_cnv_report.pdf
 coverage_qc_report: scout/demo/cancer_coverage_qc_report.html
 
-
-# meta data
-rank_model_version: '1.1'
```

Comment:
This was an easy fix. I don't think prod cancer cases have this line anyway, but I noticed the recurrent error while testing locally and I figured that we could fix it. Also for the sake of showing how cases are supposed to work!

Reply:
Hm, it probably actually existed at that time, but then changed name in Balsamic to 0.1. Anyway, good, let's be rid of it. I think we can say this closes https://github.com/Clinical-Genomics/reference-files/issues/11 as well - that one doesn't have stay open. 😊